devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy

```diff
@@ -191,6 +197,18 @@ private Map getContactResult(value) {
 	]
 }
 
+private Map getAnalogInputResult(value) {
+    Float f = Float.intBitsToFloat(value.intValue())
+    int pc = f.round(0)
```

Comment:
Since You call this method with the integer
`map = getAnalogInputResult(Integer.parseInt(descMap.value,16))`

Those operations seem to be unnecessary.

Reply:
This Type is DataType.FLOAT4. So i think need to conversion.

 example) 5 people count
   * 0000A040 ->  Integer.parseInt(descMap.value,16) ->  1084227584  ->  Float.intBitsToFloat(value.intValue()) -> 5

_